### DISCUSSION TITLE
How can I control the amount of submitted FP tasks ?

### DISCUSSION THREAD

**Discussion by DingChangjie** [deepmodeling/dpgen#466] [Q&A] [answered]
During the Labeling process, sometimes a relatively large amount of frames will be labeled (for example, there are 51 candidate frames in my first iteration). To some extent this is a burden on a many-users-shared HPC system. Except for decreasing the "fp_task_max" in one iteration, is there any way to control the amount of submitted tasks ? (i.e. let the 51 candidates be calculated _in turn_, so that they will not consume too much computing resources _concurrently_. )
Thanks !

**Comment by njzjz** [comment, marked as answer]
Adjust `fp_group_size`:
https://github.com/deepmodeling/dpgen/blob/cc816d5b93a26d82d49ab583042fa06b61645443/examples/machine/DeePMD-kit-1.0/machine-pbs-gaussian.json#L58
